glyphs2fontir/src/source.rs

```diff
@@ -0,0 +1,44 @@
+use std::{collections::HashSet, fs, path::PathBuf};
+
+use fontir::error::Error;
+use fontir::source::{Input, Paths, Source, Work};
+
+use crate::plist::Plist;
```

Comment:
I think this module should be renamed `openstep_plist` to avoid confusions with the `plist` crate which we use to parse xml plists

Reply:
Good idea.

Reply:
Done